frontend/src/pages/experiments/screens/Metrics/RunMetrics.tsx

```
@@ -0,0 +1,234 @@
+                            label={metricName}
+                          />
+                        </MenuItem>
+                      ))}
+                    </MenuContent>
+                  </Menu>
+                }
+                isVisible={isMetricDropdownOpen}
+                onDocumentClick={(event) => {
+                  if (
+                    event &&
+                    event.target &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)